R/dsl.R

```diff
@@ -431,14 +431,12 @@ menu_divider <- function(...) {
 #'
 #' @param data data to list; data.frame with fields
 #' header, icon, description
```

Comment:
please add to docs that icon and description are optional and regerete them using `devtools::document()`